app/Actions/Place/PlaceCollectionResponse.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Hedonist\Actions\Place;
+
+use Hedonist\Entities\Place\Place;
+use Illuminate\Database\Eloquent\Collection;
+
+class PlaceCollectionResponse
+{
+    private $id;
+    private $creatorEmail;
+    private $category;
+    private $city;
+    private $longitude;
+    private $latitude;
+    private $zip;
+    private $address;
+
+    public function __construct(Collection $place)
+    {
+        $this->id = $place->id;
+        $this->creatorEmail = $place->creator->email;
```

Comment:
don't use DB methods inside DTO (`$place->creator->email`). You retrive creator from model place, that means you make additional request to DB, all request must be done inside repository, so you can retrieve all data from repository in Action and then put all data into DTO